### ISSUE TITLE
Env variable not resolved at runtime

### ISSUE THREAD

**Issue by stingus** [symfony/assetic-bundle#443]
Starting with Symfony 3.3.7, the Compiler dumps the following in the project container:
```php
protected function getAssetic_Filter_SassService()
{
    $this->services['assetic.filter.sass'] = $instance = new \Assetic\Filter\Sass\SassFilter('env_sass_bin_28a320b76011ab87bad13a6c72f2aff7', '/usr/bin/ruby');
    ...
}
```
As you can see, the first parameter (which should be the sass bin path) is set with a string and not `$this->getEnv('sass_bin')`.

parameters.yml contains:
```yaml
sass_bin: '%env(sass_bin)%'
env(sass_bin): /usr/local/bin/sass
```
and config.yml:
```yaml
assetic:
  filters:
    sass:
      bin: '%sass_bin%'
```

This generates the following error when doing assetic:dump:
```
/usr/bin/ruby: No such file or directory -- env_sass_bin_28a320b76011ab87bad13a6c72f2aff7 (LoadError)
```
which is correct, since it's using the value dumped in the project container.
For now, the only solution is to hardcode the `/usr/local/bin/sass` value for the sass_bin parameter, which is not the desired behaviour.

I have no clue why this happens only for assetic starting with Symfony 3.3.7, other parameters are set in the container with `$this->getEnv('other_param')` as they should.

**Comment by chalasr** [comment]
Can you try 3.3.8 ? 3.3.7 was broken on this part.

**Comment by stingus** [comment]
@chalasr Yes, tried it, same behaviour. Actually I encountered this problem after upgrading from 3.3.6 to 3.3.8. Then I downgraded to 3.3.7 to check where this bug was introduced.

**Comment by stingus** [comment]
Fixed in Symfony 3.3.9
